src/controller/java/src/chip/devicecontroller/model/NodeState.java

```diff
@@ -34,8 +34,7 @@ public Map<Integer, EndpointState> getEndpointStates() {
     return endpoints;
   }
 
-  // Called from native code only, which ignores access modifiers.
-  private void setDataVersion(int endpointId, long clusterId, long dataVersion) {
+  public void setDataVersion(int endpointId, long clusterId, long dataVersion) {
```

Comment:
please set this as protected, so that ReportCallbackJni.Java can still can it from same package, and other package cannot see this.

Reply:
To solve this problem, we modified the NodeState method to be called directly from jni.